main/ertp/api/issuer.md

```diff
@@ -89,8 +89,7 @@ const quatloosIssuerAllegedName = quatloosIssuer.getAllegedName();
 ## `issuer.getAssetKind()`
 - Returns: `{AssetKind}`
 
-Get the kind of this `issuer`'s asset. It returns one of
-`AssetKind.NAT` (`nat`) or `AssetKind.SET` (`set`).
+Return the kind of the `issuer`'s asset; either `AssetKind.NAT` ("nat") or `AssetKind.SET` ("set").
```

Comment:
To anticipate a request, I think we'll want to document `AssetKind.COPY_SET` and `AssetKind.COPY_BAG` here, as was done in https://github.com/Agoric/documentation/commit/85fc794f2ae83a44045458d9a94c53e96074447d#diff-567699eff5882f854f56340cee0537815d3e363b61e098208212dc172190dd3dR43 . But I think that belongs in a followup.

Reply:
#665